express-api/src/controllers/users/usersController.ts

```diff
@@ -110,6 +113,38 @@ export const submitUserAccessRequest = async (req: Request, res: Response) => {
     req.body.Position,
     req.body.Note,
   );
+  const config = getConfig();
+  const user = await userServices.getUser(req.user.preferred_username);
+  const adminUsers = await userServices.getUsers({
+    role: 'Administrator',
+    agencyId: user.AgencyId,
+  });
+  const adminEmails = adminUsers.map((user) => user.Email);
+  while (adminEmails.join(';').length > 500) {
+    adminEmails.shift();
+  }
```

Comment:
Is this how the old PIMS was deciding who to send these requests to? I thought it went to a shared inbox for Lindsay/Josh.